service/src/main/kotlin/fi/espoo/evaka/messaging/bulletin/BulletinNotificationEmailService.kt

```diff
@@ -129,35 +129,49 @@ class BulletinNotificationEmailService(
 
         return when (language) {
             Language.en -> "New bulletin in eVaka$postfix"
-            Language.sv -> "Ny$postfix"
+            Language.sv -> "Ny meddelande i eVaka$postfix"
             else -> "Uusi tiedote eVakassa$postfix"
         }
     }
 
+    private fun getCitizenMessagesUrl() = if (System.getenv("VOLTTI_ENV") == "prod") "https://espoonvarhaiskasvatus.fi/messages"
```

Comment:
Couple of comments regarding this:
- How does this work in Tampere?
- How about swedish domain; should the URL be https://esbosmabarnspedagogik.fi/messages if certain condition is met (i.e. swedish speaking citizen etc.?). Maybe pass language as a parameter for this function?

We can probably proceed with this implementation but these will bite back later :)

Reply:
Yes I wished to take the easy way out but let's do this properly now 🤡  

Just opened an PR to read base url from env.